go/mysql/conn.go

```diff
@@ -1122,14 +1113,22 @@ func (c *Conn) handleNextCommand(handler Handler) error {
 		c.recycleReadPacket()
 		if err := c.writeErrorPacket(ERUnknownComError, SSUnknownComError, "command handling not implemented yet: %v", data[0]); err != nil {
 			log.Errorf("Error writing error packet to %s: %s", c, err)
-			return err
+			return false
 		}
 	}
 
-	return nil
+	return true
 }
 
-func (c *Conn) execQuery(query string, handler Handler, more bool) error {
+type execResult byte
```

Comment:
Style nit -- I'd move this type declaration up to the top of the file since `execSuccess` and `connErr` are referred to above.